### PR TITLE
Extractors feature - failing code coverage supplemented

### PR DESCRIPTION
**Pull request by tombujok**
* code coverage supplemented
* exception types unified

### CODE REVIEW

**Comment by Donnerbart** [comment]
:+1:

**Comment by jerrinot** [comment]
:+1:

File: hazelcast/src/test/java/com/hazelcast/config/MapAttributeConfigReadOnlyTest.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package com.hazelcast.config;
+
+import org.junit.Test;
+
+import static org.mockito.Mockito.mock;
+
+public class MapAttributeConfigReadOnlyTest {
```
Comment (jerrinot) [inline hazelcast/src/test/java/com/hazelcast/config/MapAttributeConfigReadOnlyTest.java]: missing test categories

Comment (Donnerbart) [inline hazelcast/src/test/java/com/hazelcast/config/MapAttributeConfigReadOnlyTest.java]: That's a good one! I should start drinking coffee before doing reviews in the morning :)

Comment (jerrinot) [inline hazelcast/src/test/java/com/hazelcast/config/MapAttributeConfigReadOnlyTest.java]: that's why there are always 2 reviewers! 

(apparently the two reviewers should be in different time-zones as well to make sure at least one of them has a normal level of caffeine)

Comment (tombujok) [inline hazelcast/src/test/java/com/hazelcast/config/MapAttributeConfigReadOnlyTest.java]: haa, I just wanted to add it realising it myself and I see that both of you have already reviewed it ;)

Comment (tombujok) [inline hazelcast/src/test/java/com/hazelcast/config/MapAttributeConfigReadOnlyTest.java]: Fixed

**Comment by jerrinot** [comment]
:+1:

**Pull request merged**